test/integration/connect/envoy/main_test.go

```diff
@@ -102,3 +107,57 @@ func discoverCases() ([]string, error) {
 	sort.Strings(out)
 	return out, nil
 }
+
+
+// crlf convert functions
+
+func travel_dir(path string ){
```

Comment:
Delete empty space before the parenthesis
Add empty space after parenthesis

`func travel_dir(path string) {`